client/dogs/FormPage.js

```diff
@@ -65,23 +67,46 @@ export default class IndexPage extends React.Component {
   constructor(props: Props) {
     super(props);
 
-    this.state = {
-      firstName: '',
-      lastName: '',
-      phone: '',
-      email: '',
-      address: '',
-      neighborhood: '',
-      zip: '',
-      dogName: '',
-      dogAge: '',
-      color: '',
-      breed: '',
-      sex: '',
-      rabiesIssued: '',
-      rabiesExpire: '',
-      comments: '',
-    };
+    if (props.results && props.results.length) {
+      const result = props.results[0];
+      this.state = {
+        firstName: result['firstName'] || '',
+        lastName: result['lastName'] || '',
+        phone: result['phone'] || '',
+        email: '',
+        address: result['address'] || '',
+        neighborhood: result['neighborhood'] || '',
+        zip: result['zip'] || '',
+        dogName: result['dogName'] || '',
+        dogYear: result['yearsOld'] || '',
```

Comment:
OK, now that I see where this is coming from, should definitely use “Old” in the names of these state properties, because just `dogYear` sounds like it could be the year the dog was born.